src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetApplicableAssetsFromPackageReports.cs

```diff
@@ -79,13 +85,13 @@ public override bool Execute()
 
             CompileAssets = compileAssets.ToArray();
             RuntimeAssets = runtimeAssets.ToArray();
+            BuildProjects = buildProjects.Distinct().Select(bp => bp.ToItem()).ToArray();
```

Comment:
What do you expect to do with the BuildProjects? Wouldn't we also need to know the set of global properties needed if we were going to rebuild them?